src/Request/ConsoleRequest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Bugsnag\Request;
+
+class ConsoleRequest implements RequestInterface
+{
+    /**
+     * The unformated console command.
+     *
+     * @var array
+     */
+    protected $command;
+
+    /**
+     * Create a new console request instance.
+     *
+     * @param array     $commandArray An array of the command line input
```

Comment:
Alignment bug. :)

Reply:
Is this something StyleCI could lint for?

Reply:
Not yet, but it is planned. I think it's actually been planned for a very long time, since I think you mentioned this feature last year too. :P

Reply:
Haha, explains the deja vu I felt when I typed that :p